### PR TITLE
ROC-2974: Remove app/* alias and update all imports

### PR DESCRIPTION
**Pull request by ellie-harrison**
### JIRA link

https://tools.hmcts.net/jira/browse/ROC-2974

### Change description
Removed TS alias "app/*" and updated all imports to follow a consistent style.

### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [x] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -12,91 +12,101 @@
...
       ],
       "drafts/*": [
-        "app/drafts/*"
+        "main/app/drafts/*"
       ],
       "claims/*": [
-        "app/claims/*"
+        "main/app/claims/*"
+      ],
+      "app-common/*": [
```
Comment (timja-kainos) [inline tsconfig.json]: what do you think about using either `common` or `shared` instead of `app-common` I'm not a big fan of the combined name?

Comment (ellie-harrison) [reply tsconfig.json]: `common` would have conflicts with another alias, but `shared` would certainly be a better name.

Comment (geek-soft) [reply tsconfig.json]: Happy with `shared` to start with. In the future we would like to remove app directory so the need for having both `common` and `shared` might disappear.

**Comment by ellie-harrison** [comment]
I have updated the `app-common` to be `common` (as it is the most commonly used of the two directories called common) and the conflicting `common` to be `shared`.

Additionally I have added 2 new aliases for the files that were imported using 'main' to tidy those up a bit.

**Pull request merged**